src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -99,7 +99,7 @@ def getWMAgentConfig(self, agentName):
         return agentConfig[0]
 
     def postWMAgentConfig(self, agentName, agentConfig):
-
+        self["requests"].delete('wmagentconfig/%s' % agentName)
```

Comment:
Oh I meant we will have separate delete method and only used in deployment script.  That way if anyone use postAgentConfig assuming there is no config for that, won't be accidentally overwriting the config.

Reply:
Wise Seangchan!!!